feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -99,7 +129,9 @@ fun RepairView() {
                 ),
                 title = stringResource(id = R.string.label_category),
                 items = optionsCategory.toList(),
-                selected = {/* TODO - Category field - To verify if extra functionality required */ }
+                selected = {
+                    repairViewModel.getCategoryFromView(it)
```

Comment:
we need to use hight order function here and send the event to the top

Reply:
Modified! Thank you for your help